server/views/index.ejs

```diff
@@ -91,7 +90,62 @@
   <div id='root'></div>
   <% if (metadata.jsonLd) { %>
     <script id="json-ld" type="application/ld+json"><%- JSON.stringify(metadata.jsonLd) %></script>
-  <% } %>  
-
+  <% } %>
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```

Comment:
```suggestion
  <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
```

Enable [Subresource Integrity (SRI) checking](https://www.w3.org/TR/SRI/) for jQuery